commands/doit.go

```diff
@@ -127,7 +127,6 @@ func configHome() string {
 // Execute executes the current command using DoitCmd.
 func Execute() {
 	if err := DoitCmd.Execute(); err != nil {
-		fmt.Println(err)
 		os.Exit(-1)
```

Comment:
I'm concerned this would hide errors we would want to return to the user.
What if we did something like this instead:

```suggestion
		if !strings.Contains(err.Error(), "unknown command") {
			fmt.Println(err)
		}
		os.Exit(-1)
```